src/native/shared/CMakeLists.txt

```diff
@@ -0,0 +1,77 @@
+set(LIB_NAME xa-shared-bits)
```

Comment:
Similar to the `runtime-base` question, what is `xa-shared-bits`?

Reply:
…and is there a good way to know when this is for a static lib vs. a shared lib vs.…

Reply:
Similar to `runtime-base`, it's a library that's shared by even more components than `runtime-base`